firmware/Makefile

```diff
@@ -9,6 +9,7 @@ endif
 BUILD := build
 BUILD_DIR = $(BUILD)/$(PROJECT)/$(PLATFORM)
 PLATFORM_DIR := projects/$(PROJECT)/platforms/$(PLATFORM)
+PLATFORM_FAMILY := $(shell echo $(patsubst stm32f%,%,$(PLATFORM)) | cut -c1)
```

Comment:
This variable is not stable. `PLATFORM` is just the name of the folder under `PROJECT/platforms/PLATFORM`. It is not necessarily the stm board name so I don't want to make this assumption.

All of our boards are stm32f767, so just bake that into the command. We'll update it in the future if needed.